frontend/src/modules/member/components/member-selection-dropdown.vue

```diff
@@ -60,6 +62,16 @@ const fetchFn = async (query, limit) => {
     return m.id !== props.id
   })
 }
+
+const totalFn = async () => {
+  try {
+    const response = await MemberService.list({}, '', 1, 0)
+
+    return response.count
+  } catch (e) {
+    return null
+  }
+}
```

Comment:
This is only called once, no?

Reply:
Yes this is only called in the `created` lifecycle of the `autocomplete-one-input` component, so it will only be called the first time the component is rendered